packages/@tinacms/toolkit/src/packages/react-sidebar/components/SidebarBody.tsx

```diff
@@ -100,6 +103,12 @@ export const FormsView = ({
           {!isMultiform && activeForm.label && (
             <FormHeader activeForm={activeForm} />
           )}
+          {formMetas &&
+            formMetas.map((meta) => (
+              <React.Fragment key={meta.name}>
+                <meta.Component />
+              </React.Fragment>
+            ))}
```

Comment:
Here's where the `formMetas` are actually being inserted into the `SidebarBody`.  I originally had them placed in the `FormBuilder`, but they were appearing in _all_ forms (Global and Overlay).